compute.tf

```
@@ -0,0 +1,71 @@
+resource "digitalocean_droplet" "this" {
```

```suggestion
resource "digitalocean_droplet" "main" {
```

We prefer `main` over `this`